### PR TITLE
Fixes #36492 - duplicate content warning for composite CVs for non-docker repos

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
Adds a notice to composite CVs publish screen if component CV versions have the same repositories added.
Screenshot from 2023-09-18 13-37-43

#### What are the testing steps for this pull request?
1, Create 2 CV versions with same repo
2.Add both to a composite CV
3. Publish. You should see a notice explaining duplicate content will be merged into the Composite CV version. The text is copied from earlier: https://github.com/Katello/katello/pull/8198/files

### CODE REVIEW

**Comment by sjha4** [comment]
Test failures expected. Will update those.

**Comment by sjha4** [comment]
Updated tests..Should be green..

File: webpack/scenes/ContentViews/Publish/CVPublishForm.js [hunk 1]
```diff
@@ -61,6 +65,24 @@ const CVPublishForm = ({
...
                 </Alert>)
             }
+            {!duplicateReposAlertDismissed && composite &&
+                (duplicateRepos !== null || duplicateRepos.length > 0) &&
+                (
+                <Alert
+                  ouiaId="duplicate-repos-alert"
+                  variant="info"
+                  isInline
+                  title={__('Duplicate repositories in component content views')}
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Publish/CVPublishForm.js:75]:
```suggestion
                  title={__('Duplicate repositories in content view versions')}
```

File: webpack/scenes/ContentViews/Publish/CVPublishForm.js [hunk 2]
```diff
@@ -61,6 +65,24 @@ const CVPublishForm = ({
...
+                  actionClose={
+                    <AlertActionCloseButton
+                      onClose={() => setDuplicateReposAlertDismissed(true)}
+                    />
+                        }
+                  style={{ marginBottom: '24px' }}
+                >
+                  <TextContent>{__('Duplicate repositories between the component content view versions will merge, resulting in a composite content view with all the content that exists among the duplicates.')}</TextContent>
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Publish/CVPublishForm.js:83]:
```suggestion
                  <TextContent>{__('Repositories in common between the content view versions will merge, resulting in a composite content view with all the content that exists among the duplicates.')}</TextContent>
```

**Review by jeremylenz — CHANGES_REQUESTED** [review]
We don't use the term "component content views" any more..

File: webpack/scenes/ContentViews/Publish/CVPublishForm.js (hunk 2 above)
Comment (jeremylenz) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js:83]: What do we mean by "all the content that exists among the duplicates?" Does this mean that if one CV version has something filtered out in a repository, but the other doesn't, I'll get the unfiltered content?

Comment (sjha4) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js]: Ya..So a union of content in the 2 repository versions between the 2 CVs..

Comment (sjha4) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js]: The message comes from the old UI which was here: https://github.com/Katello/katello/pull/8198/files

Comment (jeremylenz) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js]: Hmm.. How about
```
The composite content view will contain a combination of all the repositories from each of the content view versions.
```

Comment (sjha4) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js]: The composite content view will contain a combination of _all the content_ from the repositories from each of the content view versions.

Comment (jeremylenz) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js]: We decided on
```
Repositories common to the selected content view versions will merge, resulting in a composite content view that is a union of all content from each of the content view versions.
``

Comment (sjha4) [reply webpack/scenes/ContentViews/Publish/CVPublishForm.js]: Updated to the freshly baked text..

**Review by jeremylenz — APPROVED** [review]
Works as expected :+1: 

APJ

**Pull request merged**